impl/src/main/java/eu/darkbot/impl/managers/GalaxySpinner.java

```diff
@@ -20,15 +20,13 @@
 public class GalaxySpinner implements GalaxySpinnerAPI {
     private final byte[] BUFFER = new byte[1024];
 
-    private final HeroAPI hero;
     private final BackpageAPI backpage;
     private final GalaxyInfoImpl galaxyInfo;
     private final EventBrokerAPI eventBroker;
 
     private long lastUpdate;
 
-    public GalaxySpinner(HeroAPI hero, BackpageAPI backpage, EventBrokerAPI eventBroker) {
-        this.hero = hero;
+    public GalaxySpinner(/*keep it?*/HeroAPI hero, BackpageAPI backpage, EventBrokerAPI eventBroker) {
```

Comment:
```suggestion
    public GalaxySpinner(BackpageAPI backpage, EventBrokerAPI eventBroker) {
```
If not needed, we shouldn't keep it.
If anyone wants to use this they should be getting it via dep-injection, not calling directly, so if it breaks because they call the constructors themselves it's their fault